### ISSUE TITLE
compile STAREMaster w/o hdf4 and hdfeos

### ISSUE THREAD

**Issue by NiklasPhabian** [SpatioTemporal/STAREmaster#107]
Just an idea; but might it be worthwhile to give users the option to compile STAREMaster without hdf4 and hdfeos and just with hdf5?  hdf4 and hdfeos seem to be legacy that will eventually fade out and might not be required by all users and seem to be causing the majority of the complication during building?

**Comment by captainkirk99** [comment]
Yes, agree, but right not all the files we can read are HDF4, so if I built it without HDF4, it could not do anything. ;-)

But eventually, it will handle GOES files, which are not HDF4. At that time, it will make sense to have a build which does not require the pesky HDF4 libraries.
